### ISSUE TITLE
yaml files carry too much logic

### ISSUE THREAD

**Issue by javierbertoli** [saltstack-formulas/postgres-formula#125]
The formula's yaml files are carrying too much logic for files that must let you, at a quick glance, understand what values are being set as default or overriden in each of those files.

Even more, some of logic in these files, which should be loaded before pillars, are querying pillar values to construct variables or include values.

My understanding is that these files should just contain variables' assignments like [here](https://github.com/saltstack-formulas/tomcat-formula/blob/master/tomcat/osmap.yaml), have as little jinja as possible (like simple `set`s to avoid repetition) and then let `map.jinja` do all the `magic`.

**Comment by iggy** [comment]
Agreed... This really isn't in the spirit that we originally had in putting some of this stuff into yaml to begin with. The original "shoving everything into map.jinja" way was impossible to read. This has just come full circle to be impossible to read again.
